# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: eg@gaute.vetsj.com-20130408085359-zuz2bvzzlmqj7yqq
# target_branch: http://bazaar.launchpad.net/~catfish-search/catfish-\
#   search/trunk/
# testament_sha1: 51ca1376645ddf1ccc0bdfb97ab120a3de3da8aa
# timestamp: 2013-04-08 10:54:16 +0200
# base_revision_id: launchpad_translations_on_behalf_of_catfish-\
#   search-20130406055337-iergowyhvw6cy018
# 
# Begin patch
=== modified file 'catfish/CatfishWindow.py'
--- catfish/CatfishWindow.py	2013-03-01 20:47:39 +0000
+++ catfish/CatfishWindow.py	2013-04-08 08:53:59 +0000
@@ -50,8 +50,9 @@
 def application_in_PATH(application_name):
     """Return True if the application name is found in PATH."""
     for path in os.getenv('PATH').split(':'):
-        if application_name in os.listdir(path):
-            return True
+        if os.path.isdir (path):
+            if application_name in os.listdir(path):
+                return True
     return False
 
 def is_file_hidden(filename):

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWTCPCpIAAXZfgARUUGP/91rA
BIC////wUAOc8oKYQADJEJPMlNppoGTT1PSMjSMmmRowMBoAABoAAAAABgNAAANAAAAAAGiZKn6m
0p+npJPSfqmTIwTIaaYRoMCSQCDQmk8gSn6ZE0xT1B6mjyj01FE/Aat9/iH30whwwLOQA5QVO+lE
ewIOKP3d7QMGf1uElhI3Y3m4TuF7byOS8eZ70sw2xqqQhwhPB22Ni4aycZDeX+P4NS0d7smE+5Z9
j1HzYJuFg3v7pR4nyuOqWzbhfCsRat13MnsIkniHZ9V0cvhhbhMutiiqFNYPrIOyaaDuztK8r4gV
lurTzsu2BQQhna4oj99JMS7WoPCqqsol+ZuUSmrn8dqFTgy0ZbyzqnliiqkMZKQR3fQuXCOvK9+j
CMTCsjCOaxWDhPbQdTaya+wksKFulXUkTbstW7JBVRbA7DN8n8lPCzYpqRqFZ7Y1sjRXwUhaYq4j
c1y4bTPFKt8CVBlGAxUauy3FWR7lJ3ysyqenceUFrdcnY4/X2FbAWhnOxWqUhcuDy8+wxaD0WsCN
CpjaqoIIbBNmJYhMi4mNYFJA9xHBolJBOjcvU8rMjI8W6f16eqW62gzDe8+MGqet63rGT2Bnq0Ix
MFm8Mzibob/KC9Aat3mGly8MjGM2WzosBhEYuI/UCytTwe1ROW+cWFHCmmbwdM5qBcA2M6bA3YLR
E9pbsXqvknZWXRKUZ9UF4AbiMFdFmQQ+8bDEU2rLSuRQ1iTRZEcHNVGCF82NZK91XsUrD2QnWMtz
cdQlVLIBdQQlrGZ1SpDM4OCV9oDk5zOR/RQpHkobac7jm5u7SiVEdJqEVx78E77nRdy5w6FDf3Vq
Bv0ecR70v+S+yG2qZrUGBzOYJKnrAP7US9TJ02Nf+P1TBL4y+OZnopET3D6+DLP537oaBaFV09sA
HBqf7l32QZLyFyW8OfaEZOcmTLJQW5fa7F+ft9xXeAH89VWhWpnOECgozaIGWE71EHMsfRTedXKK
zRBky3cL/Oq1QSTkx2Mo/lphQMHLIryWq0h/C1cy5nGJQt9m030mu0VBdBcOwo3BXh++1RB7I/N/
dmUDjNZWP2deBOf0ymMQx9FDvnDXaGTxM5qvE0xHKxRpJmcDM222Lm8B5sAdZmeEwWWVw8I2BasE
z3yqlBVdw3LKXO5WYuA4dBXrRKF4S5q8Wa3p45ayDDgmOq8U7wFtmKuhsFbyFgL/xdyRThQkDCPC
pIA=
